com.woltlab.wcf/package.xml

```diff
@@ -51,11 +51,28 @@
 	</instructions>
 	
 	<instructions type="update" fromversion="5.3.*">
-		<instruction type="file" />
+		<!--
+tar cvf com.woltlab.wcf/files_pre.tar -C wcfsetup/install/files/ \
+	acp/update_5.4_session_1_cookies.php \
+	acp/update_5.4_session_2_user_session.php \
+	lib/util/HeaderUtil.class.php \
+	lib/system/package/plugin/AbstractPackageInstallationPlugin.class.php \
+	lib/system/package/plugin/ScriptPackageInstallationPlugin.class.php \
+	lib/system/package/plugin/FilePackageInstallationPlugin.class.php
+		-->
+		<instruction type="file" run="standalone">files_pre.tar</instruction>
+		<instruction type="script" run="standalone">acp/update_5.4_session_1_cookies.php</instruction>
+		<instruction type="script" run="standalone">acp/update_5.4_session_2_user_session.php</instruction>
```

Comment:
This must be part of files_pre.tar, because otherwise we're going to get quite a few ugly errors in the frontend if the community is accessed during the upgrade.